blocks/v2-video/v2-video.css

```diff
@@ -94,8 +102,12 @@ main .section.v2-video-container .v2-video-wrapper {
 }
 
 @media (min-width: 1200px) {
+  .section + .section.v2-video-container {
+    margin-top: -136px;
```

Comment:
I'm a bit afraid that if the value is changed in styles, we will forget the one in Video block. Maybe we could create variables for everything? Wdyt?